src/throttle/index.ts

```diff
@@ -42,13 +50,18 @@ export function throttle<T>({
   );
 
   // It's ok - nothing will ever start unless source is triggered
-  const $payload = createStore<T>(null as unknown as T, { serialize: 'ignore' }).on(
-    source,
-    (_, payload) => payload,
-  );
+  const $payload = createStore<T>(null as unknown as T, {
+    serialize: 'ignore',
+  }).on(source, (_, payload) => payload);
 
   const triggerTick = createEvent<T>();
 
+  const $leading = toStoreBoolean(leading, '$leading', false);
```

Comment:
In `lodash` both `leading` and `trailing` are `true` by default. Why not make the same?

Reply:
I didn't want to change the default behavior making it breaking change